### PR TITLE
feat(meshtls): initial docs

### PR DESCRIPTION
**Pull request by slonka**
rendered: https://deploy-preview-1893--kuma.netlify.app/docs/dev/policies/meshtls/

closes https://github.com/kumahq/kuma-website/issues/1890

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: app/_src/policies/meshtls.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: MeshTLS
+---
```
Comment (lahabana) [inline app/_src/policies/meshtls.md:3]: Should we add a guide along with this named something like: "Progressively rolling in strict MTLS" and then we could link it from this page?

Something that's not clear is how clients get influenced by the setting of MeshTLS. We had a discussion about this earlier this week, we should really document how this works.

File: app/_src/policies/meshtls.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+{% warning %}
+This policy uses new policy matching algorithm.
+{% endwarning %}
+
+This policy enables {{site.mesh_product_name}} to configure TLS mode, ciphers and version. 
+Backends and default mode values are taken from [the Mesh object](/docs/{{ page.version }}/policies/mutual-tls/).
```
Comment (lahabana) [inline app/_src/policies/meshtls.md:10]: Should you talk about this policy in the page above? Seems like it no?

File: app/_src/policies/meshtls.md (hunk 1 above)
Comment (slonka) [reply app/_src/policies/meshtls.md:3]: created a separate issue for this https://github.com/kumahq/kuma-website/issues/1897

Comment (slonka) [reply app/_src/policies/meshtls.md:3]: > Something that's not clear is how clients get influenced by the setting of MeshTLS. We had a discussion about this earlier this week, we should really document how this works.

In line https://github.com/kumahq/kuma-website/pull/1893/files#diff-9149b3f0d9714aeffecfdede60cbc8469e9593e7acee44002e0d0aa61444b201R35 I'm mentioning that we configure both server and client but I'll make it more explicit.

File: app/_src/policies/meshtls.md (hunk 2 above)
Comment (slonka) [reply app/_src/policies/meshtls.md:10]: added a link in `policies/mutual-tls`

File: app/_src/policies/meshtls.md (hunk 1 above)
Comment (slonka) [reply app/_src/policies/meshtls.md:3]: added something like this: https://github.com/kumahq/kuma-website/pull/1893/files#diff-9149b3f0d9714aeffecfdede60cbc8469e9593e7acee44002e0d0aa61444b201R40

File: app/_src/policies/mutual-tls.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 title: Mutual TLS
 ---
 
+{% if_version gte:2.9.x %}
+{% tip %}
+If you want to configure version, ciphers or per service permissive / strict mode check out [`MeshTLS`](/docs/{{ page.version }}/policies/mesh)
```
Comment (lahabana) [inline app/_src/policies/mutual-tls.md]:
```suggestion
If you want to configure version, ciphers or per service permissive / strict mode check out [`MeshTLS`](/docs/{{ page.version }}/policies/meshtls)
```

File: app/_src/policies/meshtls.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+---
+title: MeshTLS
+---
+
+{% warning %}
+This policy uses new policy matching algorithm.
```
Comment (lobkovilya) [inline app/_src/policies/meshtls.md:6]: Do we really need this warning? We're using it in other policies so people don't combine them with legacy twin policies (i.e. Retry and MeshRetry). But MeshTLS is kind of unique so I think we don't need it.

File: app/_src/policies/meshtls.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+{% policy_yaml example2 %}
+```yaml
+type: MeshTLS
+name: strict-mode
+mesh: default
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      kuma.io/service: redis_kuma-demo_svc_6379
```
Comment (lobkovilya) [inline app/_src/policies/meshtls.md:81]: Should we use `app: redis` tags? I think it's more future-proof than `kuma.io/service` tag

File: app/_src/policies/meshtls.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
+---
+title: MeshTLS
+---
+
+{% warning %}
+This policy uses new policy matching algorithm.
+{% endwarning %}
+
```
Comment (slonka) [inline app/_src/policies/meshtls.md:8]:
```suggestion
```

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**